components/movementsensor/gpsnmea/gpsnmea.go

```diff
@@ -32,28 +31,24 @@ type Config struct {
 
 // SerialConfig is used for converting Serial NMEA MovementSensor config attributes.
 type SerialConfig struct {
-	SerialPath               string `json:"serial_path"`
-	SerialBaudRate           int    `json:"serial_baud_rate,omitempty"`
-	SerialCorrectionPath     string `json:"serial_correction_path,omitempty"`
-	SerialCorrectionBaudRate int    `json:"serial_correction_baud_rate,omitempty"`
+	SerialPath     string `json:"serial_path"`
+	SerialBaudRate int    `json:"serial_baud_rate,omitempty"`
```

Comment:
Let's start keeping track of docs tickets for config changes we're want. I think this one should be simple enough that it should be a separate ticket. Also, announce this on breaking changes.